From 9d04c9afdb574c33f3a6602b115bd157d0e3c87f Mon Sep 17 00:00:00 2001
From: Krakenied <Krakenied1@gmail.com>
Date: Thu, 4 Apr 2024 19:19:47 +0200
Subject: [PATCH] Remove EOL message


diff --git a/proxy/src/main/java/net/md_5/bungee/BungeeCordLauncher.java b/proxy/src/main/java/net/md_5/bungee/BungeeCordLauncher.java
index c7d13591..9ed00f4b 100644
--- a/proxy/src/main/java/net/md_5/bungee/BungeeCordLauncher.java
+++ b/proxy/src/main/java/net/md_5/bungee/BungeeCordLauncher.java
@@ -66,7 +66,7 @@ public class BungeeCordLauncher
         BungeeCord bungee = new BungeeCord();
         ProxyServer.setInstance( bungee );
         bungee.getLogger().info( "Enabled Waterfall version " + bungee.getVersion() );
-        bungee.getLogger().warning("Waterfall has reached end of life and is no longer maintained. We recommend you transition to Velocity (https://papermc.io/software/velocity). For more information, see the announcement (https://forums.papermc.io/threads/1088/)."); // Waterfall - eol message
+        //bungee.getLogger().warning("Waterfall has reached end of life and is no longer maintained. We recommend you transition to Velocity (https://papermc.io/software/velocity). For more information, see the announcement (https://forums.papermc.io/threads/1088/)."); // Waterfall - eol message // Waterfall-Continued - remove EOL message
         bungee.start();
 
         if ( !options.has( "noconsole" ) )
-- 
2.45.2.windows.1

