From c3ebde06159667bbe851da32b8779994c3e4f05b Mon Sep 17 00:00:00 2001
From: Krakenied <Krakenied1@gmail.com>
Date: Fri, 26 Apr 2024 18:36:53 +0200
Subject: [PATCH] Fix modules updater


diff --git a/proxy/src/main/java/net/md_5/bungee/module/JenkinsModuleSource.java b/proxy/src/main/java/net/md_5/bungee/module/JenkinsModuleSource.java
index 4110bdc3..8824ae19 100644
--- a/proxy/src/main/java/net/md_5/bungee/module/JenkinsModuleSource.java
+++ b/proxy/src/main/java/net/md_5/bungee/module/JenkinsModuleSource.java
@@ -19,6 +19,9 @@ public class JenkinsModuleSource implements ModuleSource
         ProxyServer.getInstance().getLogger().info( "Attempting to Jenkins download module " + module.getName() + " v" + version.getBuild() );
         try
         {
+            // Waterfall-Continued start - fix modules updater
+
+            /*
             final String url = String.format(
                 "https://api.papermc.io/v2/projects/%1$s/versions/%2$s/builds/%3$s/downloads/%4$s-%2$s-%3$s.jar",
                 "waterfall",
@@ -27,6 +30,25 @@ public class JenkinsModuleSource implements ModuleSource
                 module.getName()
             );
             URL website = new URL( url );
+            */
+
+            String buildNumber = version.getBuild();
+            String moduleName = module.getName().replace('_', '-');
+            String serverVersion = net.md_5.bungee.api.ProxyServer.getInstance().getVersion().split(":")[2];
+
+            String jenkinsUrl = "https://ci.ecocitycraft.com/job/"
+                    + "Waterfall-Continued%20(Krakenied)/"
+                    + buildNumber + "/"
+                    + "io.github.waterfallmc$waterfall-module-" + moduleName + "/"
+                    + "artifact/"
+                    + "io.github.waterfallmc/"
+                    + "waterfall-module-" + moduleName + "/"
+                    + serverVersion + "/"
+                    + "waterfall-module-" + moduleName + "-" + serverVersion + ".jar";
+
+            URL website = new URL( jenkinsUrl );
+            // Waterfall-Continued end
+
             URLConnection con = website.openConnection();
             // 15 second timeout at various stages
             con.setConnectTimeout( 15000 );
-- 
2.45.1.windows.1

