From 6b469d2a71b7392ff3107abbdb470f08676ed67b Mon Sep 17 00:00:00 2001
From: Krakenied <krakenied1@gmail.com>
Date: Thu, 17 Jul 2025 18:57:46 +0200
Subject: [PATCH] BungeeCord moved Property to another package


diff --git a/proxy/src/main/java/net/md_5/bungee/ServerConnector.java b/proxy/src/main/java/net/md_5/bungee/ServerConnector.java
index 04853fb0..5b49c447 100644
--- a/proxy/src/main/java/net/md_5/bungee/ServerConnector.java
+++ b/proxy/src/main/java/net/md_5/bungee/ServerConnector.java
@@ -120,7 +120,7 @@ public class ServerConnector extends PacketHandler
 
             // Waterfall start - properties
             // Handle properties.
-            net.md_5.bungee.protocol.Property[] properties = new net.md_5.bungee.protocol.Property[0];
+            net.md_5.bungee.protocol.data.Property[] properties = new net.md_5.bungee.protocol.data.Property[0];
 
             if ( profile != null && profile.getProperties() != null && profile.getProperties().length > 0 )
             {
@@ -130,13 +130,13 @@ public class ServerConnector extends PacketHandler
             if ( user.getForgeClientHandler().isFmlTokenInHandshake() )
             {
                 // Get the current properties and copy them into a slightly bigger array.
-                net.md_5.bungee.protocol.Property[] newp = Arrays.copyOf( properties, properties.length + 2 );
+                net.md_5.bungee.protocol.data.Property[] newp = Arrays.copyOf( properties, properties.length + 2 );
 
                 // Add a new profile property that specifies that this user is a Forge user.
-                newp[newp.length - 2] = new net.md_5.bungee.protocol.Property( ForgeConstants.FML_LOGIN_PROFILE, "true", null );
+                newp[newp.length - 2] = new net.md_5.bungee.protocol.data.Property( ForgeConstants.FML_LOGIN_PROFILE, "true", null );
 
                 // If we do not perform the replacement, then the IP Forwarding code in Spigot et. al. will try to split on this prematurely.
-                newp[newp.length - 1] = new net.md_5.bungee.protocol.Property( ForgeConstants.EXTRA_DATA, user.getExtraDataInHandshake().replaceAll( "\0", "\1"), "" );
+                newp[newp.length - 1] = new net.md_5.bungee.protocol.data.Property( ForgeConstants.EXTRA_DATA, user.getExtraDataInHandshake().replaceAll( "\0", "\1"), "" );
 
                 // All done.
                 properties = newp;
-- 
2.45.2.windows.1

