From 97a312ad692b8c8317dde1344565d7cb9d6a2c80 Mon Sep 17 00:00:00 2001
From: Krakenied <Krakenied1@gmail.com>
Date: Tue, 14 May 2024 04:02:35 +0200
Subject: [PATCH] Don't send packets queued if connection has been closed


diff --git a/proxy/src/main/java/net/md_5/bungee/UserConnection.java b/proxy/src/main/java/net/md_5/bungee/UserConnection.java
index 5fda0b3f..5c8938ca 100644
--- a/proxy/src/main/java/net/md_5/bungee/UserConnection.java
+++ b/proxy/src/main/java/net/md_5/bungee/UserConnection.java
@@ -195,6 +195,12 @@ public final class UserConnection implements ProxiedPlayer
 
     public void sendPacketQueued(DefinedPacket packet)
     {
+        // Waterfall-Continued start - don't send packets queued if connection has been closed
+        if (ch.isClosed()) {
+            bungee.getLogger().log(Level.WARNING, "Could not send packet queued for " + this.name);
+            return;
+        }
+        // Waterfall-Continued end
         Protocol encodeProtocol = ch.getEncodeProtocol();
         if ( !encodeProtocol.TO_CLIENT.hasPacket( packet.getClass(), getPendingConnection().getVersion() ) )
         {
-- 
2.44.0.windows.1

