From acdd5e736461c5fec3d7cbf68274fc19c1c451ab Mon Sep 17 00:00:00 2001
From: Krakenied <krakenied1@gmail.com>
Date: Mon, 30 Jun 2025 17:37:34 +0200
Subject: [PATCH] Do not use javax annotations


diff --git a/proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java b/proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java
index 264cb648..0e3be785 100644
--- a/proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java
+++ b/proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java
@@ -214,7 +214,7 @@ public class UpstreamBridge extends PacketHandler
         // Waterfall start
         return handleChat(message, null);
     }
-    private String handleChat(String message, @javax.annotation.Nullable ClientCommand clientCommand)
+    private String handleChat(String message, /* @javax.annotation.Nullable */ ClientCommand clientCommand) // Waterfall-Continued - Do not use javax annotations; removed from guava
     // Waterfall end
     {
         for ( int index = 0, length = message.length(); index < length; index++ )
-- 
2.45.2.windows.1

